<launch>

  <node pkg="lanoise_pp" type="lanoise_pp.py" name="lanoise" output="screen" respawn="true">
    <param name="umal1_range_path" value="$(find lanoise_pp)/weights/2022-10-18-10:27:05_-lr_0.01.pth"/>
    <param name="umal2_range_path" value="$(find lanoise_pp)/weights/2022-10-29-23:54:59_-lr_0.01.pth"/>

    <!-- uncomment here for custom topic names
    <remap from="/velodyne_points" to="/your_lidar_topic"/>
    <remap from="/filtered_points" to="/noisy_lidar_topic"/>
    -->
  </node>
  
</launch> 
